permuta/MeshPatt.py

```diff
@@ -272,16 +265,16 @@ def shade(self, positions):
 
         return MeshPatt(self.pattern, self.shading | positions)
 
-    def add_point(self, box, shade_dir=DIR_NONE, safe=True):
-        """Returns a mesh pattern with a point added in box.  If shade_dir is
+    def add_point(self, pos, shade_dir=DIR_NONE, safe=True):
```

Comment:
We need to eventually go through and refactor the variable names like we did with Perm last year. I don't think that time is now though, but it needs to happen eventually.